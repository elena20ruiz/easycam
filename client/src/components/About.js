import React from 'react';
import { Card, CardTitle, CardText, CardActions, Button } from 'react-mdl';
import { Link } from 'react-router-dom';

export default () => (
  <Card shadow={0} style={{ textAlign: 'center' }}>
    <CardTitle style={{ margin: 'auto' }}>PWA</CardTitle>
    <CardText>
      About this Application
    </CardText>
    <CardActions border>
      <Link to="/"><Button colored>OK</Button></Link>
    </CardActions>
  </Card>
);