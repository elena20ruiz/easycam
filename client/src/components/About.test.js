jest.mock('react-router-dom/Link');

import React from 'react';
import renderer from 'react-test-renderer';

import About from './About';

it('About has correct contents', () => {
  const component = renderer.create(
		<About />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});