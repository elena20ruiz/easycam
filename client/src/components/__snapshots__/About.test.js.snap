exports[`test About has correct contents 1`] = `
<div
  className="mdl-card mdl-shadow--2dp"
  style={
    Object {
      "textAlign": "center",
    }
  }>
  <div
    className="mdl-card__title"
    style={
      Object {
        "margin": "auto",
      }
    }>
    <h2
      className="mdl-card__title-text">
      PWA
    </h2>
  </div>
  <div
    className="mdl-card__supporting-text">
    About this Application
  </div>
  <div
    className="mdl-card__actions mdl-card--border" />
</div>
`;
